configure.ac

```diff
@@ -932,6 +936,38 @@ if test -z "$enable_fast_no_strict_alignment" ; then
     AC_DEFINE(NEEDS_STRICT_ALIGNMENT,1,[Define if strict alignment memory access is required])
 fi
 
+if test "$enable_fast_avx_instr" = "yes" ; then
+    AC_CACHE_CHECK([whether -mavx is supported], pac_cv_found_avx,
+                   [PAC_C_CHECK_COMPILER_OPTION([-mavx],pac_cv_found_avx=yes,pac_cv_found_avx=no)],
+                   pac_cv_found_avx=no,pac_cv_found_avx=yes)
+    if test "$pac_cv_found_avx" = "yes" ; then
+        PAC_APPEND_FLAG([-mavx],[CFLAGS])
+        PAC_APPEND_FLAG([-mavx],[WRAPPER_CFLAGS])
```

Comment:
one more nitpick here, do we need to add this to WRAPPER_CFLAGS? that'll mean it gets added to `mpicc` which isn't really necessary since this code will all be in the library.

Reply:
Fair enough. I just pushed that change. I won't bother re-running the tests though.